seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2157,13 +2161,17 @@
       ],
       "challengeSeed": [
         "// Example",
-        "function functionWithArgs(a, b) {",
+        "function ourFunctionWithArgs(a, b) {",
         "  console.log(a - b);",
         "}",
-        "functionWithArgs(10, 5); // Outputs 5",
+        "ourFunctionWithArgs(10, 5); // Outputs 5",
         "",
         "// Only change code below this line.",
         "",
+        "",
+        "// Only change code above this line",
+        "",
+        "functionWithArgs(10, 5); // Should Output 10",
```

Comment:
@ChadKreutzer @raisedadead same with the comment below. This function call `functionWithArgs(10, 5)` would throw an error, I believe, because this function is not defined yet. These four lines appear to be unnecessary.